soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3947,14 +3945,18 @@ void DrawRandoEditor(bool& open) {
                 ImGui::PushItemWidth(-FLT_MIN);
 
                 //Starting Age
-                //Disabled when Forest is set to Closed
-                bool disableRandoStartingAge = !CVar_GetS32("gRandomizeForest", 0);
-                const char* disableRandoStartingAgeText = "This option is disabled because \"Forest\" is set to \"Closed\".";
+                //Disabled when Forest is set to Closed or under very specific conditions
+                //RANDOTODO: Replace magic number checks with enums
+                bool disableRandoStartingAge = (CVar_GetS32("gRandomizeForest", 0) == 0) || //If Forest is closed...
+                    (CVar_GetS32("gRandomizeDoorOfTime", 0) == 0) && //or Door of Time is Closed
+                    (CVar_GetS32("gRandomizeShuffleOcarinas", 0) == 0) && //and ocarinas are not shuffled
+                    (CVar_GetS32("gRandomizeLogicRules", 0) == 0); // and logic is glitchless
+                const char* disableRandoStartingAgeText = "This option is disabled due to other options making the game unbeatable.";
```

Comment:
i'm trying to think of a way to be more specific here ("other options" is pretty vague) but nothing is coming to mind

Reply:
I originally had it as "if Forest is closed or if DoT is closed and ocarinas aren't shuffled in glitchless logic as adult" but it really is a mouthful, considering that so far, this is the only other set of conditions that would make it impossible for adult.

Reply:
yeah, mostly commented to see if anyone else has any great ideas. i can't think of anything short enough to put in a tooltip either